testing/test_pytest_html.py

```diff
@@ -323,6 +323,42 @@ def pytest_runtest_makereport(item, call):
         src = 'data:image/png;base64,{0}'.format(content)
         assert '<img src="{0}"/>'.format(src) in html
 
+    def test_extra_jpg_image(self, testdir):
```

Comment:
Could you make `test_extra_image` take parameters instead of creating new tests? I won't block on that, though.